assets/templates/partials/count.tmpl

```diff
@@ -3,5 +3,9 @@
 <h2>
     {{ $numberResults := len .Data.Response.Items}}
     <span class="search__summary__count">{{ .Data.Response.Count}} </span>
-    {{ localise .Data.TermLocalKey $lang $numberResults }}
```

Comment:
This probably isn't working because the welsh translations are missing from the other counts (likely 2 and 3; but I can't tell from the PR what the key is to look)